packages/runtime/test/transport/files.test.ts

```diff
@@ -95,6 +96,14 @@ describe("File upload", () => {
         expect(response).toBeSuccessful();
     });
 
+    test("can upload a file with undefined as expiry date and automatically set expiry date to maximum value", async () => {
+        const response = await transportServices1.files.uploadOwnFile(await makeUploadRequest({ expiresAt: undefined as unknown as string }));
+        expect(response).toBeSuccessful();
+
+        const file = response.value;
```

Comment:
In general, use the comparison semantics of datetime objects, not their string representations. Strings representations come in various formats with different precisions, time zone offsets etc.

```ts
const defaultDate = CoreDate.from("9999-12-31T00:00:00.000Z")
expect(CoreDate.from(file.expiresAt).isSame(defaultDate)).toBe(true)

// Alternatively, you can write a custom assertion for CoreDate here:
// packages/runtime/test/customMatchers.ts
// Maybe something like isSameDate
```